src/primitives/VirtualList.tsx

```diff
@@ -0,0 +1,137 @@
+import * as s from 'solid-js';
+import * as lng from '@lightningtv/solid';
+import * as lngp from '@lightningtv/solid/primitives';
+import { List } from '@solid-primitives/list';
+import * as utils from '../utils.js';
+
+const rowOnLeft = lngp.handleNavigation('left');
+const rowOnRight = lngp.handleNavigation('right');
+const rowScroll = lngp.withScrolling(true);
+
+const rowStyles: lng.NodeStyles = {
+  display: 'flex',
+  gap: 30,
+  transition: {
+    x: {
+      duration: 250,
+      easing: 'ease-out',
+    },
```

Comment:
Why not use the user-defined default?
```suggestion
    x: true,
```